packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -486,14 +509,21 @@ async function validateGossipAttestationNoSignatureCheck(
       ? (indexedAttestationContent as electra.IndexedAttestation)
       : (indexedAttestationContent as phase0.IndexedAttestation);
 
-  const attestation: allForks.Attestation = attestationOrCache.attestation
-    ? attestationOrCache.attestation
-    : {
-        aggregationBits,
```

Comment:
miss `committeeBits` yesterday which caused the serialization issue @nflaig

Reply:
is there a way to make this more typesafe?
I'll make an issue for later, but there are some patterns in our code that lead to this kind of bug (that we should clean up later).

Reply:
yeah need to review carefully and address when we merge electra to unstable I guess @wemeetagain